src/paperless_mail/admin.py

```diff
@@ -105,5 +106,33 @@ class MailRuleAdmin(admin.ModelAdmin):
     ordering = ["order"]
 
 
+class ProcessedMailAdmin(admin.ModelAdmin):
+    class Meta:
+
+        model = ProcessedMail
+        fields = "__all__"
+
+    list_display = ("rule", "processed", "status", "subject", "received")
```

Comment:
```suggestion
    list_display = ("subject", "status", "processed", "received", "rule")
```

Again, obviously very minor, how about reordering them

Reply:
Fixed by making the subject clickable instead. the UID is pretty meaningless on its own and there's no point to display it.

Reply:
yes sure re uid, this is to re-order them